index.html

```diff
@@ -54,5 +54,165 @@ <h6>LET'S CONNECT</h6>
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap3.png" alt="" />
+        </a>
+        <ul class="leftblock">
+          <li .primarytext>
+            <h1>Tonic</h1>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap4.png" alt="" />
+        </a>
+        <ul class="leftblock2">
+          <li .primarytext>
+            <h1>Multi-Post Stories</h1>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+    </section>
+    <section class="about">
+      <h1>
+        About <br />
+        Myself
+      </h1>
```

Comment:
- [ ]  I would suggest you please use hx tags like h2,h3, and h4 because we use the h1 tags once per web page. You should not use the h1 tag twice on a single page due to semantic reason